server/test/controllers/admin/AdminImportControllerTest.java

```diff
@@ -1874,4 +1893,101 @@ public void hxSaveProgram_noDuplicatesEnabled_addsAnEmptyStatus() {
+          + "        \"translations\" : {\n"
+          + "          \"en_US\" : \"Screen 1\"\n"
+          + "        },\n"
+          + "        \"isRequired\" : true\n"
+          + "      },\n"
+          + "      \"localizedDescription\" : {\n"
+          + "        \"translations\" : {\n"
+          + "          \"en_US\" : \"Screen 1\"\n"
+          + "        },\n"
+          + "        \"isRequired\" : true\n"
+          + "      },\n"
+          + "      \"repeaterId\" : null,\n"
+          + "      \"hidePredicate\" : null,\n"
+          + "      \"optionalPredicate\" : null,\n"
+          + "      \"questionDefinitions\" : [ {\n"
+          + "        \"id\" : 2,\n"
+          + "        \"optional\" : true,\n"
+          + "        \"addressCorrectionEnabled\" : false\n"
+          + "      } ]\n"
+          + "    } ],\n"
+          + "    \"statusDefinitions\" : {\n"
+          + "      \"statuses\" : [ ]\n"
+          + "    },\n"
+          + "    \"programType\" : \"DEFAULT\",\n"
+          + "    \"eligibilityIsGating\" : true,\n"
+          + "    \"acls\" : {\n"
+          + "      \"tiProgramViewAcls\" : [ ]\n"
+          + "    },\n"
+          + "    \"categories\" : [ ], \n"
+          + "    \"localizedSummaryImageDescription\" : null\n"
+          + "  },\n"
+          + "  \"questions\" : [ {\n"
+          + "    \"type\" : \"name\",\n"
+          + "    \"config\" : {\n"
+          + "      \"name\" : \"Name\",\n"
+          + "      \"description\" : \"The applicant's name\",\n"
+          + "      \"questionText\" : {\n"
+          + "        \"translations\" : {\n"
+          + "          \"am\" : \"ስም (የመጀመሪያ ስም እና የመጨረሻ ስም አህጽሮት ይሆናል)\",\n"
+          + "          \"ko\" : \"성함 (이름 및 성의 경우 이니셜도 괜찮음)\",\n"
+          + "          \"lo\" : \"ຊື່ (ນາມສະກຸນ ແລະ ຕົວອັກສອນທຳອິດຂອງນາມສະກຸນແມ່ນຖືກຕ້ອງ)\",\n"
+          + "          \"so\" : \"Magaca (magaca koowaad iyo kan dambe okay)\",\n"
+          + "          \"tl\" : \"Pangalan (unang pangalan at ang unang titik ng apilyedo ay"
+          + " okay)\",\n"
+          + "          \"vi\" : \"Tên (tên và họ viết tắt đều được)\",\n"
+          + "          \"en_US\" : \"Please enter your first and last name\",\n"
+          + "          \"es_US\" : \"Nombre (nombre y la inicial del apellido está bien)\",\n"
+          + "          \"zh_TW\" : \"姓名（名字和姓氏第一個字母便可）\"\n"
+          + "        },\n"
+          + "        \"isRequired\" : true\n"
+          + "      },\n"
+          + "      \"questionHelpText\" : {\n"
+          + "        \"translations\" : { },\n"
+          + "        \"isRequired\" : false\n"
+          + "      },\n"
+          + "      \"validationPredicates\" : {\n"
+          + "        \"type\" : \"name\"\n"
+          + "      },\n"
+          + "      \"id\" : 1,\n"
+          + "      \"universal\" : false,\n"
+          + "      \"primaryApplicantInfoTags\" : [ ]\n"
+          + "    }\n"
+          + "  } ]\n"
+          + "}";
```

Comment:
(non-blocking) Do you mind swapping this to the """ based [multiline string](https://www.baeldung.com/java-multiline-string)? It'll be easier to read and maintain.

Reply:
done!